modules/carto/src/api/maps-v3-client.ts

```diff
@@ -338,16 +338,17 @@ async function _fetchMapDataset(
   credentials: CloudNativeCredentials,
   clientId?: string
 ) {
-  const {connectionName: connection, columns, geoColumn, source, type} = dataset;
+  const {connectionName: connection, columns, format, geoColumn, source, type} = dataset;
   // First fetch metadata
   const {url, mapFormat} = await _fetchDataUrl({
+    clientId,
     credentials: {...credentials, accessToken},
     connection,
     columns,
+    format,
     geoColumn,
     source,
-    type,
-    clientId
```

Comment:
Why is clientId removed? If we do that we'll lose the tracing at metrics

Reply:
It's not removed, I just moved it to the top of the params list to have it alphabetical

Reply:
My bad :)